prow/flagutil/github.go

```diff
@@ -66,6 +68,10 @@ func (o *GitHubOptions) Validate(dryRun bool) error {
 		}
 	}
 
+	if _, err := url.ParseRequestURI(o.gitEndpoint); err != nil {
```

Comment:
Can you also add test file to test validation of github options ?

Reply:
Sure :smile: